src/Services/HeadlessH5PService.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace EscolaLms\HeadlessH5P\Services;
+
+use Illuminate\Support\ServiceProvider;
+
+use H5PFrameworkInterface;
+use H5PFileStorage;
+use H5PStorage;
+use H5PCore;
+use H5PValidator;
+use Illuminate\Http\UploadedFile;
+
+
+use EscolaLms\HeadlessH5P\Repositories\H5PRepository;
+use EscolaLms\HeadlessH5P\Repositories\H5pDefaultFileStorage;
+use EscolaLms\HeadlessH5P\Services\Contracts\HeadlessH5PServiceContract;
+
+class HeadlessH5PService implements HeadlessH5PServiceContract
+{
+    private H5PFrameworkInterface $repository;
+    private H5PFileStorage $fileStorage;
+    private H5PCore $core;
+    private H5PValidator $validator;
+    private H5PStorage $storage;
+
+    public function __construct()
+    {
+        $this->repository = new H5pRepository();
```

Comment:
We must here use dependency injection.
Bind all those classes in ServiceProvider, add them interface, and provide them required parameters inside binding.

It may be done even by one class, that provide all required inst. Otherwise we will not be able test that class using mocking, or refactor it in future.

Reply:
Can't be done with `h5p` classes :( 

Or i don't understand this. 

Yet i think `HeadlessH5PService` is all we need for Dependecy Injection, rest of the classes will be used only in the service